.nuspec/Xamarin.Forms.AppLinks.nuspec

```diff
@@ -16,14 +16,21 @@
       <group targetFramework="MonoAndroid81">
         <dependency id="Xamarin.GooglePlayServices.AppIndexing" version="60.1142.1"/>
       </group>
+      <group targetFramework="MonoAndroid90">
+        <dependency id="Xamarin.GooglePlayServices.AppIndexing" version="60.1142.1"/>
```

Comment:
This seems redundant.  I am guessing that the MonoAndroid81 TF will not be assumed, and that is why this is listed explicitly?

Reply:
I'd need to test if it picks up MonoAndroid81 (it should) 

My other thinking had to do with the nuget listing and that it might be preferred to have it listed under a MonoAndroid90 section